src/homework.ts

```diff
@@ -13,31 +14,40 @@ export const addTwoNumbers = (a: number, b?: number): string => {
 //  * @param values - An array of numbers or strings.
 //  * @returns The sum of the numbers.
 //  */
-export const sumArray = (numbers: (number | string)[]): string => {
-  return numbers.reduce((acc, curr) => acc + curr, 0);
+export const sumArray = (numbers: (number | string)[]): number => {
+  //return numbers.reduce((acc, curr) => acc + curr, 0);
+  return Number(numbers.reduce((acc, curr) => Number(acc) + Number(curr), 0));
```

Comment:
You shouldn't have to wrap the whole thing in `Number` in order to get this to work